internal/rule.go

```diff
@@ -285,6 +276,7 @@ func (r *rule) processScanner(name string, args ...string) (chan *match, error)
 					log.Warn().Str("rule", r.name).Str("command", cmd.String()).Err(err).Msg("failed to scan command output")
 				}
 				wg.Done()
+				stop <- true
```

Comment:
This might block if the process is crashed. In that case, both stdout and stderr go routine will try writing while no one will drain the channel, resulting in one of them being stuck and thus to ressource leak. Correct way: put it in a select with default case.